src/app/services/storage.service.ts

```diff
@@ -240,4 +240,16 @@ export class StorageService {
 
     return path.indexOf('/') < 0;
   }
+
+  // Takes a string with fuzzy units like 20Mib, 20k, 20Gb, returns number in bytes
+  convertHumanStringtoNum(str: string) {
+    if(!str) { return 0 }; // if empty, return the default 0, which is unlimited
+    const letters = str.replace( /[^a-zA-Z]/g, '');
```

Comment:
The match should be that a letter is only allowed after numbers.  A letter alone is not valid ("G"), and a letter before numbers is also not valid ("g10").

Reply:
This doesn't allow a letter first, it just says that the first letter that appears has to be one of our set, and if it isn't, we return NaN and stop there. The reason for adding it is that the const unit = ... regex below seems to turn any letter not in our list to a 'b'.